### PR TITLE
roberta, albert added to glue example, data caching added

### PR DESCRIPTION
**Pull request by ekmb**
Signed-off-by: Evelina Bakhturina <ebakhturina@nvidia.com>

### CODE REVIEW

File: nemo/collections/nlp/data/datasets/glue_benchmark_dataset/glue_benchmark_dataset.py [hunk 1]
```diff
@@ -68,14 +72,44 @@
...
+            ),
         )
 
+        if use_data_cache and os.path.exists(cached_features_file):
+            logging.info(f"loading from {cached_features_file}")
+            with open(cached_features_file, "rb") as reader:
+                self.features = pickle.load(reader)
+        else:
+            token_params = {
+                'bos_token': None,
```
Comment (yzhang123) [inline nemo/collections/nlp/data/datasets/glue_benchmark_dataset/glue_benchmark_dataset.py]: there should be a tokenizer.bos_token

File: examples/nlp/glue_benchmark/glue_benchmark_with_bert.py [hunk 2]
```diff
@@ -31,6 +31,8 @@
 --data_dir /path_to_data_dir/MRPC \
 --task_name mrpc \
 --work_dir /path_to_output_folder \
+--pretrained_model_name roberta-base \
```
Comment (yzhang123) [inline examples/nlp/glue_benchmark/glue_benchmark_with_bert.py]: could you make bert as default, and if you use roberta/albert add a note in the comment that there are also other alternatives?

Comment (ekmb) [reply examples/nlp/glue_benchmark/glue_benchmark_with_bert.py]: done

File: nemo/collections/nlp/data/datasets/glue_benchmark_dataset/glue_benchmark_dataset.py (hunk 1 above)
Comment (ekmb) [reply nemo/collections/nlp/data/datasets/glue_benchmark_dataset/glue_benchmark_dataset.py]: I'm not using bos here, just cls token

**Pull request merged**